Add Part 3: coding a complete shoot-em-up game (#57)
gbdev/gb-asm-tutorial: src/part3/spawning-enemies.md

# Spawning Enemies

> For spawning enemies, we increase a counter each frame. When this counter reaches a set value, we’ll TRY to spawn an enemy. If an enemy is actual spawned, we’ll reset the counter. When trying to spawn a value, we’ll get a random unsigned 8-bit number. This number will be the enemies x posittion. From: [https://github.com/pinobatch/libbet/blob/master/src/rand.z80#L34-L54](https://github.com/pinobatch/libbet/blob/master/src/rand.z80#L34-L54) 

Reviewer: ```suggestion
For spawning enemies, we increase a counter each frame. When this counter reaches a set value, we’ll TRY to spawn an enemy. If this succeds, we’ll reset the counter. When trying to spawn a value, we’ll get a random unsigned 8-bit number. This number will be the enemies x position.
```

TODO: "From: [https://github.com/pinobatch/libbet/blob/master/src/rand.z80#L34-L54](https://github.com/pinobatch/libbet/blob/master/src/rand.z80#L34-L54)" should go in a reference at the bottom

Author: i'm not sure what you mean by this..

Author: That file is now obsolete, will remove it.
